### PR TITLE
Mediawiki fix probes & improve notes

### PR DESCRIPTION
**Pull request by juan131**
**What this PR does / why we need it:**

This PR provides several improvements to Mediawiki Helm Chart:

- Fix Readiness/Liveness probes.
- Improves NOTES
- Update MariaDB dependency

### CODE REVIEW

**Review by carrodher — APPROVED** [review]

File: stable/mediawiki/templates/ingress.yaml [hunk 1]
```diff
@@ -3,10 +3,10 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | trimSuffix "-" -}}"
+  name: {{ template "mediawiki.fullname" $ }}
```
Comment (tompizmor) [inline stable/mediawiki/templates/ingress.yaml:6]: why the `$` sign? Shouldn't that be a `.` ?

File: stable/mediawiki/values.yaml [hunk 2]
```diff
@@ -226,12 +226,14 @@ resources:
 ## Configure extra options for liveness and readiness probes
 ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes)
 livenessProbe:
+  enbaled: true
```
Comment (tompizmor) [inline stable/mediawiki/values.yaml]: typo here

**Review by tompizmor — CHANGES_REQUESTED** [review]

File: stable/mediawiki/templates/ingress.yaml (hunk 1 above)
Comment (juan131) [reply stable/mediawiki/templates/ingress.yaml:6]: It's because of the range

**Review by tompizmor — APPROVED** [review]
/approve
/lgtm

**Pull request merged**